content/chapter/03.tex

```diff
@@ -194,7 +195,123 @@ \subsubsection*{Sensortechnologie}
 \end{itemize}
 
 \subsubsection*{Messung der Unwucht}
-Hier kommen dann Bilder rein und wir erklären und diskutieren die Ergebnisse dann, oder?
+Wie bereits unter dem vorherigen Punkt beschrieben, existieren verschiedene Sensoren auf dem \ac{9-DOF}.
+Für die Messung der Unwucht wird zunächst allerdings nur der Beschleunigungssensor eingesetzt.
+Die von ihm gelieferten Werte zeigen am deutlichsten eine Verschiebung des Motorblockes, aufgrund von Vibrationen, an.
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=8cm]{images/chapter/03/measurement_arduino_a2.png}
+	\caption{Ablaufdiagramm für die Unwuchtmessung am \ac{A2}}
+	\label{fig:sequence_measure_a2}
+\end{figure}
+
+Für die erste Messreihenermittlung wurde das Programm von \ac{A2} auf den Ablauf programmiert, wie in Abbildung \ref{fig:sequence_measure_a2} dargestellt.
+Hierbei wartet \ac{A2} auf einen Interrupt von \ac{A1} um mit der Unwuchtmessung zu beginnen.
+Sobald ein weiterer Interrupt an \ac{A2} gesendet wird, kann davon ausgegangen werden, dass der Propeller eine vollständige Umdrehung absolviert hat und die Unwuchtmessung abgeschlossen werden kann.
+Dieser erste Test soll zeigen, ob mögliche Muster in den Messwerten einer einzelnen Propellerumdrehung zu sehen sind.
+Verwendet wird hierfür ein Propeller, dessen Unwucht auf der gleichen Seite, wie die Markierung ist.
+
+Die aufgezeichneten Messwerte der einzelnen Messungen für diesen ersten Test, sind in zwei Tabellen dargestellt, welche jeweils drei Messergebnisse enthalten. Die Tabelle \ref{tab:rounds_balance} enthält die Messergebnisse für einen Propeller ohne eigens erstellte Unwucht.
+
+\begin{table}[H]
+\centering
+\resizebox{\textwidth}{!}{%
+\begin{tabular}{lllllllllllll}
+\textbf{Messung} & \textbf{Zeit} & \textbf{X} & \textbf{Y} & \textbf{Z} & \textbf{Zeit} & \textbf{X} & \textbf{Y} & \textbf{Z} & \textbf{Zeit} & \textbf{X} & \textbf{Y} & \textbf{Z} \\
+\hline
+1       & 7    & 1,94  & 0,11  & 10,41 & 8    & 0,56  & -0,29 & 10,10 & 9    & 4,12  & 0,46  & 11,16 \\
+2       & 15   & 2,10  & 0,86  & 10,61 & 16   & -0,24 & 0,08  & 9,54  & 17   & -0,34 & 0,39  & 9,26  \\
+3       & 24   & -2,57 & -0,29 & 8,49  & 25   & -2,11 & -0,16 & 7,55  & 25   & 1,79  & -0,09 & 10,13
+\end{tabular}%
+}
+\caption{Messergebnisse von Propellerumdrehungen ohne Unwucht}
+\label{tab:rounds_balance}
+\end{table}
+
+Aus dieser Tabelle können mehrere Erkenntnisse gezogen werden.
+Zuerst ist erkennbar, dass pro Messung (Zeit, X, Y, Z) nur drei Messpunkte aus dem Beschleunigungssensor gelesen werden konnten.
```

Comment:
Das ist ja auch in der Vergangenheit passiert, ich referenziere auf die Messergebnisse, welche ja bereits vorliegen. Das Messen ist bereits geschehen.